apps/backend/src/app/api/latest/email-templates/crud.tsx

```diff
@@ -48,6 +48,10 @@ export const emailTemplateCrudHandlers = createLazyProxy(() => createCrudHandler
     }
   },
   async onUpdate({ auth, data, params }) {
+    if (auth.tenancy.config.email_config.type === 'shared') {
+      throw new StatusError(StatusError.Forbidden, 'Cannot update email templates in shared email config. Set up a custom email config to update email templates.');
```

Comment:
403 is for auth-related stuff
```suggestion
      throw new StatusError(StatusError.BadRequest, 'Cannot update email templates in shared email config. Set up a custom email config to update email templates.');
```